app/views/components/add-another/index.html

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-one-half">
+
+      <!-- <h1 class="govuk-heading-l">MOJ Frontend</h1> -->
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "../"
+      }) }}
+
+      <h1 class="govuk-heading-xl" tabindex="-1">
+        <span class="govuk-caption-xl">Components</span> Add another
+      </h1>
+
+      <h2 class="govuk-heading-l moj-add-another__heading">Add a person</h1>
```

Comment:
This needs to be inside the component container for the component script to work. In this case . you have the component as the form below the heading.